components/pipeline-service/development/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - git::https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=c3d1e5f85e64205a3c4fd7476d0ef0d03e4b570f
+  - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd?ref=19f250cc874d49d5eb6afb56f46f8d1411e9d225
```

Comment:
```suggestion
  - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service?ref=19f250cc874d49d5eb6afb56f46f8d1411e9d225
```

Reply:
And storage component you can install using separated argocd template, like I did in the pr https://github.com/redhat-appstudio/infra-deployments/pull/1427 which you asked to close.

Reply:
But up to you.

Reply:
@AndrienkoAleksandr not sure I understood your comment, currently we use an argo app to install pipeline-service storage backends (minio and postgres). Can you better describe what you are suggesting and why it may a better choice ?

Reply:
Ok I understood the issue, we can't use kustomize to patch chains if we are importing an argocd app and not the manifests themselves. Working on a fix ..

Reply:
@AndrienkoAleksandr I understood what you did in your PR. 
That's probably the right way to do it, It is just that we don't want a specific app for minio but for both minio and postgres. I am going to fix this PR based on your approach

Reply:
@AndrienkoAleksandr @Roming22 
after discussing this with @gbenhaim, it is better to stick with the kustomize overlay and not add another application to stonesoup to install pipeline-service-storage in development environment. So I have edited this PR accordingly